index.js

```diff
@@ -1,12 +1,4 @@
-var Handlebars = require('handlebars');
-
-if (Handlebars['default']) {
```

Comment:
I'm not sure if these changes should be moved to Rendr, or where exactly they should live